scripts/install.sh

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/env sh
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reply:
We don't need to make this `sh` compatible :)

Reply:
oh, woops..! I may have pressed enter too quickly when prompted with my snippet.

Reply:
Erm, this is and always has been a POSIX sh script.
I put in a lot of effort to make it posix sh compliant, I'm don't think this is a good idea.

Reply:
Also since this is a zsh plugin manager for me it does not make sense to require bash to be installed to get it. It would only hurt users that are trying to get zinit.

Reply:
> Also since this is a zsh plugin manager for me it does not make sense to require bash to be installed to get it. It would only hurt users that are trying to get zinit.

_Most_ OSes come with bash installed already (and if you're on an OS that doesn't, you certainly know how to install it). I see POSIX `sh` compliance as a hindrance to development here, but since you're doing the work, it's really up to you,

Reply:
Most do, but not all. That's precisely the point of using posix instead of bash.
alpine, busybox etc do not ship bash for instance. 

Requiring bash now that is always worked with pure posix sh will also break some workflows.
I don't want to break any existing workflows.

Reply:
As to further development I don't see POSIX being a hindrance. The installer is pretty static, for me it is pretty much feature-complete already.